### DISCUSSION TITLE
My codespace stop working "Unable to resolve resource vscode-remote://codespace.."

### DISCUSSION THREAD

**Discussion by JuanDragovich** [community/community#57873] [Codespaces]
### Select Topic Area

General

### Body

My codespace stop working "Unable to resolve resource vscode-remote://codespace.."
![4de3f130-3557-438d-b92e-9b444f1ff323](https://github.com/community/community/assets/126341629/90534d44-f7f7-4a52-b328-bf2b7bef808d)
![7fe4a244-5852-4fc3-93fc-3bb6e3863b21](https://github.com/community/community/assets/126341629/7b605c7c-6ffb-4d15-a596-fee9ff9fc986)
Any help?

**Comment by cwndrws** [comment]
Hi 👋 thanks for writing in. Has this problem persisted?

**Comment by kukuthecoder** [comment]
yes for me
![Screenshot from 2023-08-07 13-01-22](https://github.com/community/community/assets/15101344/35469b7f-3fa8-4772-a25f-bfbece45ada4)

**Reply by mehulrajdev** [reply]
use the local host

**Comment by mehulrajdev** [comment]
use the localhost

**Comment by kukuthecoder** [comment]
i would want ti use it on the cloud. is there a way to use it without opening my visual studio code?
![Screenshot from 2023-08-07 13-53-04](https://github.com/community/community/assets/15101344/56ebb8ec-746c-4c14-af66-4727c35a2f48)

if i try open within my github repository it throws that error

**Reply by Cirruslucent** [reply]
My account won't work now Milehigh.world

**Reply by devashish2024** [reply]
You can open it on vscode for web but it seems there is currently a bug with it stuck on **Setting up your codespace** so I'd recommend and the only option is VSCode

**Reply by Cirruslucent** [reply]
Thank you
